### PR TITLE
fix names of local variables in m/*/*.eb easyconfigs

### PR DESCRIPTION
**Pull request by boegel**
required to avoid warnings when these easyconfigs are parsed because of changed made in https://github.com/easybuilders/easybuild-framework/pull/2938

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 33 out of 33 (33 easyconfigs in this PR)
gligar05.gastly.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz, Python 2.7.5
See https://gist.github.com/8619a1b8ba40513534bb5aa8276f4bc5 for a full test report.

File: easybuild/easyconfigs/m/miRDeep2/miRDeep2-2.0.0.8-intel-2016b.eb [hunk 1]
```diff
@@ -2,16 +2,16 @@ easyblock = 'PackedBinary'
...
 homepage = 'https://www.mdc-berlin.de/8551903/en/'
 description = "miRDeep2 is a completely overhauled tool which discovers microRNA genes by analyzing sequenced RNAs"
 
 toolchain = {'name': 'intel', 'version': '2016b'}
 
-source_url_subdir = 'research/research_teams/systems_biology_of_gene_regulatory_elements/projects/miRDeep/'
-source_urls = ['https://www.mdc-berlin.de/45995549/en/' + source_url_subdir]
-sources = ['mirdeep%s.zip' % altver]
+local_source_url_subdir = 'research/research_teams/systems_biology_of_gene_regulatory_elements/projects/miRDeep/'
+source_urls = ['https://www.mdc-berlin.de/45995549/en/' + local_source_url_subdir]
```
Comment (Micket) [inline easybuild/easyconfigs/m/miRDeep2/miRDeep2-2.0.0.8-intel-2016b.eb]: could just inline this variable

Comment (boegel) [reply easybuild/easyconfigs/m/miRDeep2/miRDeep2-2.0.0.8-intel-2016b.eb]: No, line too long :)

Comment (Micket) [reply easybuild/easyconfigs/m/miRDeep2/miRDeep2-2.0.0.8-intel-2016b.eb]: Well, I would certainly always prefer a + and a newline when cutting up long strings in python than this procedure, but I also don't think this old obscure config is worth fighting over.
```
source_urls = ['https://www.mdc-berlin.de/45995549/en/' + 
               'research/research_teams/systems_biology_of_gene_regulatory_elements/projects/miRDeep/']
```

**Comment by Micket** [comment]
Going in, thanks @boegel!

**Pull request merged**